api/tests/core/bookings/test_api.py

```diff
@@ -494,6 +494,24 @@ def test_no_booking_if_external_booking_fails(
             assert Booking.query.count() == 0
             assert str(exc.value) == "Something wrong happened"
 
+        @override_features(ENABLE_CDS_IMPLEMENTATION=True)
+        def test_book_manual_offer(self):
+            beneficiary = users_factories.BeneficiaryGrant18Factory()
+            cds_provider = get_provider_by_local_class("CDSStocks")
+            venue_provider = providers_factories.VenueProviderFactory(provider=cds_provider)
+            providers_factories.CinemaProviderPivotFactory(venue=venue_provider.venue)
+            manual_offer = offers_factories.EventOfferFactory(
```

Comment:
```suggestion
            non_synced_offer = offers_factories.EventOfferFactory(
```
(et dans le nom de fonction)